src/store/peer/types.ts

```diff
@@ -1,9 +1,23 @@
+import {Group} from "../group/types";
+
 export interface Peer {
+  ID?: string,
```

Comment:
Codacy has a fix for the issue: [Replace `,` with `;`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  ID?: string;
```